structlog/processors.py

```diff
@@ -207,36 +207,37 @@ class TimeStamper(object):
         <http://en.wikipedia.org/wiki/ISO_8601>`_, or `None` for a `UNIX
         timestamp <http://en.wikipedia.org/wiki/Unix_time>`_.
     :param bool utc: Whether timestamp should be in UTC or local time.
+    :param str key: Property name stored in `event_dict`.
```

Comment:
I don’t think ”Property” is the right term here.  Maybe something like “Target key in `event_dict` for added timestamps.”?